tests/integration/cmor/_fixes/icon/test_icon.py

```diff
@@ -15,7 +15,10 @@
 import esmvalcore.cmor._fixes.icon.icon
 from esmvalcore.cmor._fixes.fix import GenericFix
 from esmvalcore.cmor._fixes.icon._base_fixes import IconFix
-from esmvalcore.cmor._fixes.icon.icon import AllVars, Clwvi, Hfls, Hfss
+from esmvalcore.cmor._fixes.icon.icon import (
+                                            AllVars, Clwvi,
+                                            Hfls, Hfss, Rtmt, Rtnt
+                                            )
```

Comment:
Can you run `isort tests/integration/cmor/_fixes/icon/test_icon.py` to make sure this is properly formatted?